lib/bugsnag.rb

```diff
@@ -595,6 +562,13 @@ def unwrap_bundler_exception(exception)
 
       unwrapped.cause
     end
+
+    # Expose the feature flag delegate for {Bugsnag::Utility::FeatureDataStore}
+    #
```

Comment:
Should we annotate with `@api private` as per the declaration in `configuration`?

Reply:
It doesn't need to be because it's actually private: https://github.com/bugsnag/bugsnag-ruby/blob/4b4203ca81cd048a60d7ac94c72812075f9aacf6/lib/bugsnag.rb#L435